src/sentry/api/endpoints/project_stats.py

```diff
@@ -61,7 +63,10 @@ def get(self, request, project):
         elif stat == 'forwarded':
             stat_model = tsdb.models.project_total_forwarded
         else:
-            raise ValueError('Invalid stat: %s' % stat)
+            try:
+                stat_model = FILTER_STAT_KEYS_TO_VALUES[stat]
+            except KeyError:
+                raise ValueError('Invalid stat: %s' % stat)
```

Comment:
@tkaemming I was wondering if this implementation is strange since I'm raising a `ValueError` after `KeyError`. This should match previous behavior in that it will mean that it went through the whole list and did not find a matching value, I just didn't know if this is weird to do.